.github/workflows/main.yml

```diff
@@ -62,12 +53,15 @@ jobs:
       - name: Install python dependencies
         run: |
           pip install --user --upgrade pip
-          pip install tox
+          pip install pre-commit
+          pip install mypy==0.782
```

Comment:
```suggestion
```
I think it is safe to remove these lines because they are listed as requirements in `dev_requirements.txt`